seqr/views/apis/dataset_api.py

```diff
@@ -110,6 +112,37 @@ def add_variants_dataset_handler(request, project_guid):
     Family.bulk_update(
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
+    if project.workspace_name and project.workspace_namespace:
+        if has_analyst_access(project):
+            safe_post_to_slack(
+                SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+                """{num_sample} new samples are loaded in https://seqr.broadinstitute.org/project/{guid}/project_page
+                ```{samples}```
+                """.format(
+                    num_sample=len(matched_sample_id_to_sample_record),
+                    guid=project.guid,
+                    samples=[id for id in matched_sample_id_to_sample_record.keys()]
+                ))
+        else:
+            send_html_email(
+                """Hi {user},
+We are following up on your request to load data from AnVIL on {date}.
+We have loaded data from the AnVIL workspace “{namespace}/{name}” to the corresponding seqr project {proj_name}. {num_sample} samples are currently loaded. Let us know if you have any questions.
```

Comment:
we should have a link to the workspace and the project page. Also, lets redo the content of the email to 
```
Hi {user},
We are following up on your request to load data from AnVIL to seqr on {date}.
We have loaded {num_sample} samples from the AnVIL workspace <a>{namespace}/{name}</a> to the corresponding seqr project <a>{proj_name}</a>. 
- The seqr team
```